test/e2e/framework/metrics/api_server_metrics.go

```diff
@@ -50,3 +50,11 @@ func (g *Grabber) getMetricsFromAPIServer(ctx context.Context) (string, error) {
 	}
 	return string(rawOutput), nil
 }
+
+func (g *Grabber) getMetricsSLIsFromAPIServer(ctx context.Context) (string, error) {
```

Comment:
Why are `Grabber` methods not all in the same file?

Reply:
Yeah that is strange, thanks for bringing this to notice. Moved all Grabber methods to metrics_grabber.